src/test/java/linkedlist/SingleLinkedListUtilTest.java

```diff
@@ -16,28 +15,45 @@ public class SingleLinkedListUtilTest {
 
     @BeforeEach
     void setUp() {
-//        singleLinkedList1.insert(4);
-//        singleLinkedList1.insert(1);
+//        Node node = new Node(4);
+//        singleLinkedList1.insert(node);
+//        node = new Node(1);
```

Comment:
you should remove unused code before merge to main. 
if it needed just remove comment, unused remove all.